src/pytest_plugins/execute/execute.py

```diff
@@ -56,6 +56,18 @@ def pytest_addoption(parser):
             "unless overridden by the test."
         ),
     )
+    execute_group.addoption(
```

Comment:
Execution tests set the txn gas limit, is this because the tests have no control over the block gas limit on the testnet?